wpilibj/src/athena/java/edu/wpi/first/wpilibj/hal/SolenoidJNI.java

```diff
@@ -20,13 +20,13 @@
 
   public static native boolean getSolenoid(int portHandle);
 
-  public static native byte getAllSolenoids(byte module);
+  public static native byte getAllSolenoids(int module);
```

Comment:
It seems like this should return int, as that matches the HAL and C++ return types.